specification/compute/resource-manager/Microsoft.Compute/preview/2020-09-30/gallery.json

```diff
@@ -0,0 +1,2836 @@
+    "OSDiskImageEncryption": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/DiskImageEncryption"
+        }
+      ],
+      "description": "Contains encryption settings for an OS disk image."
+    },
+    "DataDiskImageEncryption": {
+      "properties": {
+        "lun": {
+          "type": "integer",
+          "format": "int32",
+          "description": "This property specifies the logical unit number of the data disk. This value is used to identify data disks within the Virtual Machine and therefore must be unique for each data disk attached to the Virtual Machine."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/DiskImageEncryption"
+        }
+      ],
+      "required": [
+        "lun"
+      ],
+      "description": "Contains encryption settings for a data disk image."
+    },
+    "DiskImageEncryption": {
+      "properties": {
+        "diskEncryptionSetId": {
+          "type": "string",
+          "description": "A relative URI containing the resource ID of the disk encryption set."
+        }
+      },
+      "description": "This is the disk image encryption base class."
+    },
+    "GalleryArtifactSource": {
+      "properties": {
+        "managedImage": {
+          "$ref": "#/definitions/ManagedArtifact"
+        }
+      },
+      "required": [
+        "managedImage"
+      ],
+      "description": "The source image from which the Image Version is going to be created."
+    },
+    "ManagedArtifact": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The managed artifact id."
+        }
+      },
+      "required": [
+        "id"
+      ],
+      "description": "The managed artifact."
+    },
+    "GalleryImageVersionPublishingProfile": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/GalleryArtifactPublishingProfileBase"
+        }
+      ],
+      "description": "The publishing profile of a gallery image Version."
+    },
+    "GalleryImageVersionStorageProfile": {
+      "properties": {
+        "source": {
+          "$ref": "#/definitions/GalleryArtifactVersionSource"
+        },
+        "osDiskImage": {
+          "$ref": "#/definitions/GalleryOSDiskImage"
+        },
+        "dataDiskImages": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GalleryDataDiskImage"
+          },
+          "description": "A list of data disk images."
+        }
+      },
+      "description": "This is the storage profile of a Gallery Image Version."
+    },
+    "GalleryArtifactVersionSource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The id of the gallery artifact version source. Can specify a disk uri, snapshot uri, or user image."
+        },
+        "uri": {
+          "type": "string",
+          "description": "The uri of the gallery artifact version source. Can be a disk uri, snapshot uri, or user image."
+        }
```

Comment:
`"The uri of the gallery artifact version source. Can be a disk uri, snapshot uri, or user image."`
Is this true? Are we using this for more than vhds/blobs?

Reply:
Thanks for pointing out the silly mistake!